### PR TITLE
Xenoarch scanning time halved

### PR DESCRIPTION
**Pull request by Boaz1111**
## About the PR

Halved the xenoarch scanning time
## Why / Balance

Discussed this with Emo. 
## Media

- [X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Artifact analyzer now scans twice as fast!

### CODE REVIEW

**Comment by UbaserB** [comment]
YOOOOO

File: Resources/Prototypes/Entities/Structures/Machines/artifact_analyzer.yml [hunk 1]
```diff
@@ -42,10 +42,8 @@
   - type: ApcPowerReceiver
     powerLoad: 12000
     needsPower: false #only turns on when scanning
-  - type: UpgradePowerDraw
-    powerDrawMultiplier: 0.80
-    scaling: Exponential
   - type: ArtifactAnalyzer
+    analysisDuration: 30
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Structures/Machines/artifact_analyzer.yml:46]: just change the default value of the datafield in c# if this is supposed to be the default value

**Review by mirrorcult — CHANGES_REQUESTED** [review]

**Comment by Kadeo64** [comment]
I didn't even think about this. We don't have part upgrades anymore, so there's no reason for the scan to be a minute long. Maybe scientists will actually be able to research things before power goes out.

File: Resources/Prototypes/Entities/Structures/Machines/artifact_analyzer.yml (hunk 1 above)
Comment (Boaz1111) [reply Resources/Prototypes/Entities/Structures/Machines/artifact_analyzer.yml:46]: shit good point

**Comment by ItsMeThom** [comment]
As part of the machine upgrade rework, will there be an upgraded artifact scanner to do this instead?

**Comment by EmoGarbage404** [comment]
someone is free to add that whenever.

**Comment by Kadeo64** [comment]
> As part of the machine upgrade rework, will there be an upgraded artifact scanner to do this instead?

we already have an upgraded scanner (the traversal distorter)

**Comment by deltanedas** [comment]
thats not a scanner it biases node direction

**Comment by ItsMeThom** [comment]
Yeah what deltanedas said. its useful but has to be installed on top of the xenoarch pad in order to bias the direction the artifact moves in the node tree. If its not already on the cards it would be cool to get a T2 research for xenoarch for an experimental pad design (maybe an octagon?) that boosts the scan speed at the cost of power, or leaking radiation like the experimental vessel.

**Pull request merged**